lib/module.js

```diff
@@ -21,6 +21,8 @@ module.exports = function sentry (moduleOptions) {
     publishRelease: process.env.SENTRY_PUBLISH_RELEASE || false,
     disableServerRelease: process.env.SENTRY_DISABLE_SERVER_RELEASE || false,
     disableClientRelease: process.env.SENTRY_DISABLE_CLIENT_RELEASE || false,
+    attachCommits: process.env.SENTRY_AUTO_ATTACH_COMMITS === 'true' || true,
```

Comment:
It's still not possible to override to `false`. `false || true` is still `true`.

I think it would make sense to have some helper function for handling that. Like:
```js
function getEnvironmentValueOrDefault(envKey, defaultValue) {
  if (process.env[envKey] !== undefined) {
    return process.env[envKey] === 'true'
  }

  return defaultValue
}
// ...
attachCommits: getEnvironmentValueOrDefault('SENTRY_AUTO_ATTACH_COMMITS', true),
```

Reply:
Even better would be to maybe treat `1` or `true` as truthy values and `0` or `false` as falsey. I think people often use 0/1 for env variables.

Reply:
Oh wow completely missed that. My mind was telling me that `||` is JavaScripts nullable operator but obviously it isn’t.

Reply:
Fixed now, I guess